addon/modifiers/did-insert.js

```diff
@@ -1,4 +1,4 @@
-import Ember from 'ember';
+import { setModifierManager, capabilities } from '../modifier';
```

Comment:
Would you mind updating to `ember-cli-babel@7.10.0` so we can remove this (and the underscored `_setModifierManager` usage over in `addon/modifier.js`) in favor of:

```js
import { setModifierManager, capabilities } from '@ember/modifier';
```

Reply:
@rwjblue I may be way off here, but it looks like that doesn't work prior to canary because of a typo introduced in 3.7 that was fixed in:
https://github.com/emberjs/ember.js/commit/b8a5633948348e35a1235f2b7a0f56b33d3e516a#diff-c424149274ab88ce43f2847da12c5ce0L531

It was exported under the global as `Ember._modifierManagerCapabilties` (missing "i" before "ties") so I'm thinking the mapping doesn't work until that commit...? I'm not sure how to address this other than the intermediary file solution

Reply:
😩 - Ya, good point